File: Assignment No.39/Program4.java
/*Problem Statement :Write a Program which accept one number and position from user and check whether bit at that position and toggle that bit and return modified data.
Input : 10 3
Output : 14
*/
import java.util.*;

class Program4
{
	public static int ToggleBit(int iNo,int ipos)
	{
		
		int iMask=0x00000001,iResult=0;

		iMask=iMask << (ipos-1);

		iResult=iNo ^ iMask;
		return iResult;
			 
	}
	public static void main(String args[])
	{
		int iNo = 0;
		int ipos =0;
		int iret=0;

		Scanner sobj = new Scanner(System.in);
		System.out.println("Enter the Number :");
		iNo = sobj.nextInt();

		System.out.println("Enter the Position :");
		ipos = sobj.nextInt();
		
		iret = ToggleBit(iNo,ipos);
		
		System.out.println("Modified code :"+iret);
	}
}

File: Assignment No 3/Program4.c
//Write a program which accept one character from user and convert case of that character.
//Input : a
//Output : A
#include<stdio.h>

void DisplayConvert(char cValue)
{
	if((cValue >= 'A')&&(cValue <= 'Z'))
	{
		cValue=cValue+32;
		printf("%c",cValue);
	}
	else if((cValue >= 'a')&&(cValue <= 'z'))
	{
		cValue=cValue-32;
		printf("%c",cValue);
	}
}
int main()
{
	char cValue='\0';
	printf("Enter character\n");

	scanf("%c",&cValue);
	DisplayConvert(cValue);
	return 0;
}